src/screens/LoginScreen/components/Blockimg/index.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+import img from '../../assets/img/blockimg.svg';
+
+import styles from './index.module.scss';
+
+function Blockimg() {
```

Comment:
Acordate que si el nombre de algo esta compuesto por 2 palabras o más se escribe en camelCase para que sea más fácil leerlo
```suggestion
function BlockImg() {
```

Reply:
Listo gracias